src/plugins/event-plugins/__integ__/WebVitalsPlugin.test.ts

```diff
@@ -17,24 +17,6 @@ fixture('WebVitalEvent Plugin').page(
 // "FID is not reported if the user never interacts with the page."
 // It doesn't seem like TestCafe actions are registered as user interactions, so cannot test FID
 
-const removeUnwantedEvents = (json: any) => {
```

Comment:
question: The changes to this file (`WebVitalsPlugin.test.js`) look good to me. However, are these related to fixing the integ tests? If so, maybe we should include in PR description for future reference.

Reply:
You are correct, I am pushing an unrelated change with this PR, which is generally bad practice.

The test was failing, so I refactored it while debugging. It turned out to be something else, however since the change improves the test's maintainability and is small, I didn't want to open a second PR.